beeline-core/src/main/java/io/honeycomb/beeline/tracing/propagation/CompositeHttpHeaderPropagtor.java

```diff
@@ -0,0 +1,73 @@
+package io.honeycomb.beeline.tracing.propagation;
+
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+import java.util.Optional;
+
+/**
+ * Codec that encompasses one or more codec implementations and applies each codec on
+ * each call to encode and decode.
+ *
+ * <h1>Thread-safety</h1>
+ * Instances of this class are thread-safe and can be shared.
+ */
+public class CompositeHttpHeaderPropagtor implements PropagationCodec<Map<String, String>> {
```

Comment:
Typo. 

```suggestion
public class CompositeHttpHeaderPropagator implements PropagationCodec<Map<String, String>> {
```

Reply:
Plus all the other mentions, and the test file :)

Reply:
Doh!